src/v/v8_engine/tests/executor_test.cc

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "seastarx.h"
+#include "v8_engine/executor.h"
+
+#include <seastar/core/sleep.hh>
+#include <seastar/core/when_all.hh>
+#include <seastar/testing/thread_test_case.hh>
+
+#include <algorithm>
+
+SEASTAR_THREAD_TEST_CASE(task_queue_full_test) {
```

Comment:
Can we add a test where task throws an exception ?

Reply:
We have  tests like this in v8_script_test

I can add one in executor test

Reply:
added it